docker-compose.yml

```diff
@@ -44,6 +48,12 @@ services:
         environment:
             MINIO_ACCESS_KEY: cratesfyi
             MINIO_SECRET_KEY: secret_key
+    prometheus:
+        build:
+            context: ./docker
+            dockerfile: ./Dockerfile-prometheus
+        ports:
+            - "9090:9090"
```

Comment:
Making the prometheus service optional can be done by [making an extra file](https://stackoverflow.com/a/45682432). While that's not ideal in any sense, it's the best that can be done until this [open but closed](https://github.com/docker/compose/issues/1896) issue gets through

Reply:
Sure, but now we have to document and remember that there are two different compose files. I'm not sure the maintenance is worth it since the startup time is so low.